package.json

```diff
@@ -45,6 +45,7 @@
     "@storybook/addon-actions": "^5.0.11",
     "@storybook/react": "^5.0.10",
     "autoprefixer": "^9.6.1",
+    "axios": "^0.19.0",
```

Comment:
this should be a `peerDependancy` read more about it (here)[https://nodejs.org/en/blog/npm/peer-dependencies/#using-peer-dependencies]

Reply:
both dev and peer dep

Reply:
done